src/jdk.hotspot.agent/macosx/native/libsaproc/MacosxDebuggerLocal.m

```diff
@@ -44,6 +44,12 @@
 #import <sys/ptrace.h>
 #include "libproc_impl.h"
 
+#define UNSUPPORTED_ARCH "Unsupported architecture!"
```

Comment:
Noot needed here IMO
it was done already inlibproc_impl.h for both amd64 and aarch64.

Reply:
It's used on line 520 of the file. But the main reason, it is there in master https://github.com/openjdk/jdk/blob/master/src/jdk.hotspot.agent/macosx/native/libsaproc/MacosxDebuggerLocal.m#L45

Reply:
Oh, I see your point. This line was moved to libproc_impl.h. The build was fine for me, I assumed it should not in case of duplicating defines. Thanks, I'll take a closer look.

Reply:
We needed to share this define between multiple .m/.c files hence moved to .h file